client/rpcserver/types_test.go

```diff
@@ -97,7 +97,7 @@ func TestParseNewWalletArgs(t *testing.T) {
 	for _, test := range tests {
 		nwf, err := parseNewWalletArgs(test.params)
 		if err != nil {
-			if !errors.Is(err, test.wantErr) {
+			if !errors.As(err, &test.wantErr) {
```

Comment:
This modifies `test.wantErr`.  That's pretty hard to swallow.  I think wrapping the other error value in types.go is making this awkward.  I don't think it's testing the right thing anymore either.
@JoeGruffins Can you provide feedback on the suggested changes to client/rpcserver/types.go with the other error wrapped?

Reply:
errArgs doesn't have a unique type, so yes this isn't testing what was intended anymore. Any standard error should unwrap into errArgs. I guess the linter doesn't want "%v" to be used with errors? The easy answer would be to just use the error string `err.Error()` for the second part of those errors and throw away the type. This is probably fine and won't change the current behavior. Another answer would be to give errArgs a type and an Unwrap method I think?

Reply:
Or just revert these types.go changes.

Reply:
reverted for now